client/packages/requisitions/src/ResponseRequisition/DetailView/DetailView.tsx

```diff
@@ -45,6 +57,27 @@ export const DetailView: FC = () => {
     );
   }, []);
 
+  const onProgramIndicatorClick = useCallback(
+    (
+      programIndicator?: ProgramIndicatorFragment,
+      indicatorLine?: IndicatorLineRowNode,
+      response?: ResponseFragment
+    ) => {
+      // TODO: Snack?
+      if (!response || !indicatorLine) return;
```

Comment:
Wanna add a snack to let the users know why they can't click the button if they happen to click is and there's nothing there? or remove this and disable the buttons down below